### PR TITLE
Make rubocop-emacs TRAMP-aware

### LINKED ISSUE #12: rubocop-emacs doesn't work via tramp
There is actually two issues, but they're closely related:
1. rubocop-emacs is checking rubocop installation on local machine, so we'll get 'RuboCop is not installed' if it isn't installed locally
2. rubocop-emacs uses buffer-file-name, so rubocop is run like `rubocop --format emacs /ssh:notebook:smth.rb`. Filename should be translated before call (something like [this](http://erlang.org/pipermail/erlang-patches/2013-August/004228.html))

### PR DESCRIPTION
**Pull request by ojab**
Fixes #12 (actually only pt. 2 of #12,  pt. 1 seems impractical to me).

### CODE REVIEW

File: rubocop.el [hunk 1]
```diff
@@ -58,6 +58,20 @@
   :group 'rubocop
   :type 'string)
 
+(defun rubocop-local-file-name (file-name)
```
Comment (bbatsov) [inline rubocop.el]: Add a docstring for the function.

File: rubocop.el [hunk 2]
```diff
@@ -58,6 +58,20 @@
...
+  (let ((file-name
+         (or file-name
+             buffer-file-name)))
+    (cond ((tramp-tramp-file-p file-name)
+           (rubocop-tramp-get-localname file-name))
+          (t
+           file-name))))
+
+(defun rubocop-tramp-get-localname (file-name)
```
Comment (bbatsov) [inline rubocop.el]: Here as well.

**Comment by ojab** [comment]
Merged both functions into one, docstring added.

**Comment by bbatsov** [comment]
:+1: Thanks!

**Pull request merged**